File: Algorithms/Implementation/Divisible Sum Pairs/divisibleSumPairs.cs
static int divisibleSumPairs(int n, int k, int[] ar) {
	int counter = 0;

	for(int i = 0; i < ar.Length; i++){
		for(int j = 0; j < i; j++){
			if((ar[i] + ar[j]) % k == 0){
				   counter++;
			}
		}
	}
	return counter;
}

File: Algorithms/Implementation/Divisible Sum Pairs/divisibleSumPairs.py
#Problem URL: https://www.hackerrank.com/challenges/divisible-sum-pairs/problem

def divisibleSumPairs(n, k, ar):
    counter = 0

    for i in range(0, len(ar)):
        for j in range(0, i):
            if((ar[i] + ar[j]) % k == 0):
                counter += 1

    return counter
